lib/mbta.js

```diff
@@ -2,10 +2,64 @@
 
 // Code here.
 
+// stopsBetweenStations('Red Line', 'South Station', 'Red Line', 'Alewife')
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+  let mbtaStops = {
+      'Red': [
+        'South Station',
+        'Park Street',
+        'Kendall',
+        'Central',
+        'Harvard',
+        'Porter',
+        'Davis',
+        'Alewife'
+      ],
+      'Green': [
+        'Government Center',
+        'Park Street',
+        'Boylston',
+        'Arlington',
+        'Copley',
+        'Hynes',
+        'Kenmore'
+      ],
+      'Orange': [
+        'North Station',
+        'Haymarket',
+        'Park Street',
+        'State',
+        'Downtown Crossing',
+        'Chinatown',
+        'Back Bay',
+        'Forest Hills'
+      ]
+  };
+
```

Comment:
You could declare `mbtaStops` as separate object outside of the function to keep things cleaner.